components/x-audio/src/components/Players.jsx

```diff
@@ -11,8 +11,12 @@ import { TimeRemaining } from './TimeRemaining';
 import { Marquee } from './Marquee';
 import { Image } from './Image';
 
-const Title = ({ text }) => <Marquee className={classNameMap('audio-player__info__title')} text={text} />
-const SeriesName = ({ text }) => <Marquee className={classNameMap('audio-player__info__series-name')} text={text} />
+const Title = ({ text }) => (
+	<div className={classNameMap('audio-player__title__marquee')}>
+		<Marquee className={classNameMap('audio-player__info__title')} text={text} />
+	</div>
+);
+const SeriesName = ({ text }) => <div className={classNameMap('audio-player__info__series-name')}><span title={text}>{text}</span></div>;
```

Comment:
Does the series name not need to be a marquee anymore?

Reply:
📝 The reasons why we remove the marquee from `SeriesName`
- When  both `Title` and `SeriesName`'s scrolling marquee animation happen, it is impossible to read either of them.
- Initially it was not asked by Designer.
- `SeriesName` is tend to be short. It's an edge case that `SeriesName` needs the animation.

We will put back it when requested.